configs/MSI/P35-Neo3.conf

```diff
@@ -0,0 +1,43 @@
+# MSI P35-Neo3 MS-7395
+# Values pulled from: https://www.alldatasheet.com/datasheet-pdf/pdf/257913/FINTEK/F71882FG.html
+# See "Application circuit section page 122"
+# Labels based on BIOS Hardware Monitor Names
+chip "f71882fg-isa-0a00"
+
+	# Fans
+	label fan1 "CPU Fan"
+	label fan2 "Case Fan"
+	ignore fan3 # Not present - invalid data
+	ignore fan4 # Not present  - invalid data
+
+	#Temperatures
+	label temp1 "CPU"
+	label temp2 "Auxilliary"
```

Comment:
I named it auxillary since i wasn't sure what it's actually measuring. The BIOS 'system temperature' itself is misleading as it doesn't correlate to anything in particular. I've gone over the board with a thermal probe. I can change it to System Temperature though. It just isn't  well defined/known what that 'system' temperature it's measuring is.

Reply:
Ok, I just wanted to make it consistent with other software. From the manual it seems some MSI's windows software reports it as "system temperature" as well.